deepchecks/core/check_utils/single_feature_contribution_utils.py

```diff
@@ -52,8 +52,11 @@ def pps_df_to_trace(s_pps: pd.Series, name: str):
                   )
 
 
-def get_single_feature_contribution(train_df: pd.DataFrame, train_label_name: Optional[Hashable], test_df: pd.DataFrame,
-                                    test_label_name: Optional[Hashable], ppscore_params: dict, n_show_top: int,
+def get_single_feature_contribution(train_df: pd.DataFrame, train_label_name: Optional[Hashable],
+                                    test_df: pd.DataFrame,
+                                    test_label_name: Optional[Hashable], ppscore_params: dict,
+                                    n_show_top: int,
+                                    min_pps_to_show: float = 0.05,
```

Comment:
You didn't add min_pps_to_show to the check's parameters...
To this one or to single feature

Reply:
i added to simple feature, should i add to single too?

Reply:
Yes, I think so. Also missed that it was added to the check, sorry

Reply:
added